ouster-ros/src/point_cloud_processor_factory.h

```diff
@@ -84,7 +84,7 @@ class PointCloudProcessorFactory {
     }
 
     template <typename PointT>
-    static LidarScanProcessor make_point_cloud_procssor(
+    static LidarScanProcessor make_point_cloud_processor(
```

Comment:
:man_facepalming:  Oh no, I can't believe my eyes :eyes:
Thanks for catching that.